src/frontend/api/hentAInntektUrl.ts

```diff
@@ -0,0 +1,14 @@
+import type { FamilieRequest } from '@navikt/familie-http/dist/HttpProvider';
+
+import { RessursResolver } from '../utils/ressursResolver';
+
+export async function hentAInntektUrl(request: FamilieRequest, ident: string): Promise<string> {
+    const ressurs = await request<{ ident: string }, string>({
+        method: 'POST',
+        url: 'familie-ks-sak/api/a-inntekt/hent-url',
+        data: {
+            ident: ident,
+        },
```

Comment:
Nitpick: Kan vurdere å skrive det slik:

```suggestion
        data: { ident },
```